docs/HowTo/Get-Started/Connect-To-Testnet.md

```diff
@@ -149,24 +144,32 @@ It may take more than 8 hours for a deposit to become active.
 
 ### Start the validator
 
-Run Teku and specify the validator key file [created earlier](#generate-the-validator-and-send-the-deposit).
+Run Teku and specify the validator key file [created earlier](#generate-the-validator-and-send-the-deposit),
+and the text file containing the password to decrypt the validator key.
 
 !!! example
 
     ```bash
     teku --network=medalla --eth1-endpoint=http://localhost:8545 \
-    --validators-key-files=validator_key/validator_888eeef/validator_888eeef.json \
-    --validators-key-password-files=./medalla/password.txt \
+    --validator-keys=validator/keys/validator_888eef.json:validator/passwords/validator_888eef.txt \
     --rest-api-enabled=true --rest-api-docs-enabled=true \
     --metrics-enabled
     ```
 
-!!! tip
+If you have multiple validator key files, place the key files and accompanying passwords in one or
+more directories. Use [`--validator-keys`](../../Reference/CLI/CLI-Syntax.md#validator-keys) to
+specify the directory to load the keys and passwords from.
+
+The key file and password file must have identical names when specifying directories.
```

Comment:
Identical except for the extension.  So `keys/validator1.json` matches `passwords/validator1.txt` and `keys/subdir/validator2.json` matches `passwords/subdir/validator2.txt`.